app/assets/javascripts/miq_application.js

```diff
@@ -722,6 +722,8 @@ function miqAjaxAuth(url) {
   })
   .then(null, function() {
     add_flash(__('Incorrect username or password'), 'error', { id: 'auth_failed' });
+    $('#user_name').val('');
```

Comment:
I've noticed the service UI does not clear the username, in the sake of consistency it probably makes sense to do the same or change in both. any opinions? /cc @serenamarie125